recipes/siesta/meta.yaml

```diff
@@ -0,0 +1,58 @@
+{% set name = "siesta" %}
+{% set version = "4.0.1" %}
+{% set md5 = "5cb60ce068f2f6e84fa9184ffca94c08" %}
+{% set blas_variant = "openblas" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://launchpad.net/{{ name }}/4.0/{{ version }}/+download/{{ name }}-{{ version }}.tar.gz
+  md5: {{ md5 }}
```

Comment:
Would it be possible to sha256?

Reply:
The launchpad tar's are intrinsically added a md5, although I could download and get sha256, is it really beneficial?